routing/pathfind.go

```diff
@@ -534,82 +550,99 @@ func findPath(tx *bolt.Tx, graph *channeldb.ChannelGraph,
 
 		// If this vertex or edge has been black listed, then we'll skip
 		// exploring this edge.
-		if _, ok := ignoredNodes[v]; ok {
+		if _, ok := ignoredNodes[fromVertex]; ok {
 			return
 		}
 		if _, ok := ignoredEdges[edge.ChannelID]; ok {
 			return
 		}
 
+		nodeInfo := distance[toNode]
+
+		amountToSend := nodeInfo.amountToReceive
+
+		// Compute fee based on the amount that needs to be send to the
```

Comment:
note that this is the fee "charged" by the current node.

Reply:
Comment updated